content/docs/1.5.0/deploy/upgrade/_index.md

```diff
@@ -25,10 +25,10 @@ In addition, Longhorn disallows downgrading to any previous version to prevent u
   |  x.y.*      |  x.(y-1).*  |   X    |  v1.6.0  to  v1.5.0  |
   |  x.y.*      |  x.y.(*-1)  |   X    |  v1.5.1  to  v1.5.0  |
 
-[^lastMinorVersion]: Longhorn only allows upgrades from any patch version of the last minor release before the new major version. (For example, v1.30.* is allowed to upgrade to v2.0.*, given that v1.30 is the last minor release branch before 2.0.)
+[^lastMinorVersion]: Longhorn only allows upgrades from a patch version of the last minor release before the new major version. For example, you can upgrade from v1.3.* to v2.0.* because v1.3.0 is the last minor release before v2.0.
 
-> **Warning**:
-> * Upgrade path enforcement is introduced in Longhorn v1.5.0, which means that downgrading from v1.5.0 to any previous version is possible. **Please note that downgrading is not supported**.
+> **Important:**
+> If necessary, you can downgrade from v1.5.0 to an earlier version.
```

Comment:
The `upgrade enforcement & downgrade prevention` has been introduced since 1.5.0. If users try to downgrade to any previous version < 1.5, it can't be prevented as the capability was not implemented in older versions. 

The revised sentence may give the impression that downgrading is allowed and supported, but in reality, it is not. Even though it can be carried out as described above, we must emphasize that downgrading is unsupported and we do not recommend users to do so.